beacon/goclient/events.go

```diff
@@ -58,16 +59,28 @@ func (gc *GoClient) startEventListener(ctx context.Context) error {
 		strTopics = append(strTopics, string(topic))
 	}
 
-	for _, client := range gc.clients {
-		if err := client.Events(ctx, strTopics, gc.eventHandler); err != nil {
-			gc.log.Error(clResponseErrMsg, zap.String("api", "Events"), zap.Error(err))
+	isMultiClientListener := !gc.withWeightedAttestationData
+	logger := gc.log.
+		With(zap.Int("clients_len", len(gc.clients))).
+		With(zap.String("topics", strings.Join(strTopics, ", "))).
+		With(zap.Bool("is_multi_client_listener", isMultiClientListener))
```

Comment:
i dont think there's a reason to log this because we already log the clients on creation, the topics are static at compile time, and weighted param should be logged separately before (if not already?)
```suggestion
	logger := gc.log.With(
		zap.Int("clients_len", len(gc.clients)),
		zap.String("topics", strings.Join(strTopics, ", ")),
		zap.Bool("is_multi_client_listener", isMultiClientListener))
```